ros_ws/launch/navigation_stack.launch

```diff
@@ -10,6 +10,9 @@
     <arg name="use_gpu" default="true"/>
     <arg name="joystick_type" default="xbox360"/>
     <arg name="visualize_lidar" default="false"/>
+    <arg name="pf_angle_step" default="18"/>
+	<arg name="pf_max_particles" default="4000"/>
```

Comment:
Maybe that a bit high? I get similar good results with much lower values.

Reply:
It's just the default value from the `particle_filter` package. Any suggestions what our default should be?